website/docs/guides/docs/docs-multi-instance.mdx

```diff
@@ -1,50 +1,45 @@
 ---
 id: multi-instance
-description: Use multiple docs plugin instances on a single Docusaurus site.
+description: Use the multi-instance feature to serve 2 or more doc sets.
 slug: /docs-multi-instance
 ---
 
 # Docs Multi-instance
 
-The `@docusaurus/plugin-content-docs` plugin can support [multi-instance](../../using-plugins.mdx#multi-instance-plugins-and-plugin-ids).
+The `@docusaurus/plugin-content-docs` plugin supports [multi-instance](../../using-plugins.mdx#multi-instance-plugins-and-plugin-ids). The multi-instance feature allows a site to serve 2 or more distinct sets of documentation from the same Docusaurus source.
 
 :::note
 
-This feature is only useful for [versioned documentation](./versioning.mdx). It is recommended to be familiar with docs versioning before reading this page. If you just want [multiple sidebars](./sidebar/multiple-sidebars.mdx), you can do so within one plugin.
+If you only want multiple sidebars, use a `sidebar` plugin instead. See [Using multiple sidebars](./sidebar/multiple-sidebars.mdx) for how to do this.
 
 :::
 
 ## Use-cases {#use-cases}
 
-Sometimes you want a Docusaurus site to host 2 distinct sets of documentation (or more).
+These are 2 of the everyday use cases for the multi-instance feature.
 
-These documentations may even have different versioning/release lifecycles.
```

Comment:
I still think it's mostly useful for versioned docs. Clarifying your use cases with a very concrete example first will be important.

There are certainly cases where 2 unversioned plugins make sense but these are not the most common cases

One use-case I can think of is if you have 2 doc sets for which you need significantly different plugin options.